build/compose-sepolia.yaml

```diff
@@ -0,0 +1,25 @@
+# This compose file contains the config to run the node with the Sepolia testnet.
+# The config points to the echo Python application running in Sepolia.
+# It is necessary to set the RPC provider address in the environment variable RPC_ADDRESS.
+# Notice the node will run in reader mode, so it doesn't need a Ethereum wallet.
+
+version: "3.9"
+
+name: rollups-node
+services:
+  node:
+    environment:
+      CARTESI_BLOCKCHAIN_ID: "11155111"
+      CARTESI_BLOCKCHAIN_HTTP_ENDPOINT: "https://${RPC_ADDRESS}"
+      CARTESI_BLOCKCHAIN_WS_ENDPOINT: "wss://${RPC_ADDRESS}"
```

Comment:
This won'r work with Infura, for ex.

```
Sepolia	JSON-RPC over HTTPS	https://sepolia.infura.io/v3/YOUR-API-KEY
Sepolia	JSON-RPC over WebSocket	wss://sepolia.infura.io/ws/v3/YOUR-API-KEY
```

Maybe we could use the conventional `RPC_URL` and `WS_URL` of other web3 tools.

Reply:
I changed it to RPC_HTTP_URL and RPC_WS_URL. I think we might be able to remove the RPC_HTTP_URL in the future because it is a subset of the websocket API.